src/components/loading-spinner/loading-spinner.tsx

```diff
@@ -0,0 +1,27 @@
+import { Component, h, Prop } from '@stencil/core';
+
+export type LoadingSpinnerColor = 'dark' | 'light';
+
+export type LoadingSpinnerColorMap = { [key in LoadingSpinnerColor]: string };
+
+@Component({
+  tag: 'bds-loading-spinner',
+  styleUrl: 'loading-spinner.scss',
+  shadow: true,
+})
+export class BdsLoadingSpinner {
+  /**
+   * 	Sets the color of the spinner, can be 'light' or 'dark'
+   */
+  @Prop() color: LoadingSpinnerColor = 'light';
+
+  render() {
+    return (
+      <div class="sk-circle">
+        {[...Array(12).keys()].map((_, i) => (
```

Comment:
Add to constant. What the purpose of the number 12 here?

Reply:
it's the number of divs necessary to build the spinner